staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -127,3 +134,19 @@ func TestParserV1beta1(t *testing.T) {
 		t.Errorf("Unexpected policy! Diff:\n%s", diff.ObjectDiff(policy, expectedPolicy))
 	}
 }
+
+func TestPolicyCntCheck(t *testing.T) {
+	for _, tc := range testCases {
+		// Create an illegal policy file.
+		f, err := ioutil.TempFile("", "illegalPolicy.yaml")
+		require.NoError(t, err)
+		defer os.Remove(f.Name())
+
+		_, err = f.WriteString(tc.caseContent)
+		require.NoError(t, err)
+		require.NoError(t, f.Close())
+
+		_, err = LoadPolicyFromFile(f.Name())
```

Comment:
This part is a copy-paste from another test. Extract to a separate function maybe?

Smth like `_, err := loadPolicyFromString(tc.policy)`

Reply:
Thx for review. but, LoadPolicyFromFile is the function what I'm testing in reader.go.

Reply:
I don't mind that, you will have the call to that function in the code. But if you want to have it called explicitly in the code -- that's OK also, wrap the rest at least:

```
f, err := writePolicy(tc.policy)
require.NoError(t, err)
defer os.Remove(f)

_, err = LoadPolicyFromFile(f)
```

Reply:
WriteString is a function imported from package "os" 
> func (f *File) WriteString(s string) (n int, err error)
maybe I can only change caseContent to policy?

Reply:
> WriteString is a function imported from package "os"

I don't understand how it makes extracting a function impossible. Could you please elaborate?

Reply:
What I mean is doing the following:

```
func TestPolicyCntCheck(t *testing.T) {
	for _, tc := range testCases {
		f, err := writePolicy(tc.policy)
		require.NoError(t, err)
		defer os.Remove(f)

		_, err = LoadPolicyFromFile(f)
		assert.Errorf(t, err, "loaded illegal policy with 0 rules from testCase %s", tc.caseName)
	}
}

func writePolicy(policy string) (string, error) {
		...
}
```

And replace the same piece of code in `TestParserV1beta1`

Reply:
Oh very sorry about that. I misunderstood what you meant just now.